OpenCDMi/OCDM.config

```diff
@@ -8,6 +8,9 @@ endif()
 
 map()
     kv(outofprocess ${PLUGIN_OPENCDMI_OOP})
+    if(PLUGIN_OPENCDMI_CONTAINER)
```

Comment:
Although it might work better to not have the outofprocess from the line above together with the newer "mode" keyword this could be confusing, mode is the newer version of outofprcocess with more options (e.g. outofprocess:false equals "mode":"off"). So having same in one config file might be confusing. So I would also use the if above to not include outofprocess.

Reply:
Yes, you are correct, please see correction from LGI side.

Reply:
Maybe , if you look at pull -request #1569, it is usefull to use the same semantics. Allow from the outside to define the mode, like: kv(mode ${PLUGIN_OPENCDMI_MODE})

Reply:
Updated. Please see corrections.